generamba/lib/generamba/module_generator.rb

```diff
@@ -5,88 +6,85 @@
 module Generamba
 
 	# Responsible for creating the whole code module using information from the CLI
-	class ModuleGenerator
+	class ModuleGenerator < Thor
```

Comment:
Oh, I missed two lines with the `yes` method.

I don't like the idea of using `Thor` and its functions on the business logic layer (and `ModuleGenerator`) definitely belongs to this layer. We shouldn't mess presentation and business logic in one class.

I see two possible solutions:
- You can perform this check somewhere in the `gen_command.rb` (maybe by using some helper class)
- You can post a warning, that these files are not empty, and stop the module generation process. To force overwrite, you can add some flag to `generamba gen` command, like `generamba gen MyModule MyTemplate --force`.

What do you think?